README.md

```diff
@@ -114,10 +114,22 @@ $ ./m8c
 INFO: Looking for USB serial devices.
 INFO: Found M8 in /dev/ttyACM1.
 INFO: Opening port.
-Aborted (core dumped)
+ERROR: Error: Failed: Permission denied
 ```
 
-This is likely an issue with the version of libserialport included in your Linux distribution and you need to build the library yourself. Please see [this issue for a workaround](https://github.com/laamaa/m8c/issues/20).
+This is likely caused because the user running m8c does not have permission to use the serial port. The eaiest way to fix this is to add the current user to a group with permission to use the serial port.
```

Comment:
But it's not caused by a bad version of libserialport or anything. It's just that the user doesn't have permissions to open the serial port. I updated this FAQ to explain how to configure it correctly without having to compile libserialport yourself.

Reply:
The problem that was addressed in the linked issue wasn't actually related to device permissions, but libserialport trying to use termiox with the device while Linux kernels had dropped support for it.

At least Ubuntu finally seems to have updated their packages to a version including the libserialport fix and I was unable to reproduce the original problem easily, this replaced entry might make more sense in the FAQ.